packages/components/src/FilterBar/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from "./FilterBar"
+export * from "./subcomponents"
```

Comment:
We have dot notation so that we don't export these directly 😬 What are you trying to export from these? Types?

Reply:
Oh oops I think I put this in the wrong place